scripts/stall-analyser.py

```diff
@@ -32,16 +32,20 @@
                     help='Skip common backtrace prefix terminated by one or more addresses greater or equal to the threshold (0=disabled)')
 parser.add_argument('-e', '--executable',
                     help='Decode addresses to lines using given executable')
+parser.add_argument('-f', '--full-addr2line', action='store_const', const=True, default=False,
+                    help='Decode full addr2line information, otherwise print concised report')
```

Comment:
Personally, I wouldn't (and didn't) understand what this explanation mean. What does "full addr2line" information contain that a "consise" (sic) report doesn't?

Reply:
Sure. I'll clarify it.  The concise form strips out function parameters' types using `c++filt -p`.